app/src/main/java/uk/ac/ebi/atlas/experimentpage/ExperimentFileLocationService.java

```diff
@@ -45,6 +45,8 @@ public Path getFilePath(String experimentAccession, ExperimentFileType fileType)
                 return dataFileHub.getRnaSeqBaselineExperimentFiles(experimentAccession).tpms.getPath();
             case RNASEQ_D_ANALYTICS:
                 return dataFileHub.getBulkDifferentialExperimentFiles(experimentAccession).analytics.getPath();
+            case PROTEOMICS_D_ANALYTICS:
+                return dataFileHub.getBulkDifferentialExperimentFiles(experimentAccession).analytics.getPath();
```

Comment:
We are using the same file as `RNASEQ_D_ANALYTICS` wouldn't it cause the same issue that @pcm32 arose earlier that someone might change that file without considering the consequences on proteomics experiments?

Reply:
These two files are separated now as different cases, so that we resolved the coupling issue.